polytope/polytope.py

```diff
@@ -290,6 +290,32 @@ def intersect(self, other, abs_tol=ABS_TOL):
 
         return reduce(Polytope(iA, ib), abs_tol=abs_tol)
 
+    def translation(self, d):
+        """Translate the Polytope by a given vector.
+
+        Consult L{polytope.polytope._translate} for implementation details.
+
+        @type d: 1d array
+        @param d: The translation vector.
+
+        @rtype: L{Polytope}
+        @return: A translated copy of the Polytope.
+        """
+        newpoly = self.copy()
+        _translate(newpoly, d)
+        return newpoly
+
+    def rotation(self, u, v, theta=None):
+        """Rotate the Polytope. Only simple rotations are implemented at this
```

Comment:
Yes, a more mathematically rigorous description would probably be helpful.

When I say path, I mean "the series of simple rotations required to reorient from one orientation to another." By **simple rotation**, I revise my earlier definition to "a rotation that may be described by only one plane of rotation". These simple rotations are, as you said, reorientations that may be projected into a hyperplane. It is useful to also define **compound rotation**, "a rotation that has multiple planes of rotation and is the composition of multiple simple rotations".

Although [Euler's rotation theory](https://en.wikipedia.org/wiki/Euler%27s_rotation_theorem) says that all reorientations in 3D are simple rotations, for `N > 3` this is no longer true. As you said, the parameterization limits the capability of the mapping, and I believe that the current input parameters are not capable of expressing the compound rotations. In other words, the parameters of this function are only capable of expressing simple rotations at this time.

I guess the phase "at this time" is misleading. Will this function be expanded to include compound rotations in the future? No. I think it is better that only simple rotations are implemented, and users can perform compound rotations by applying a sequence of simple rotations.

Does this make sense? I will be using this explanation to clear up the docs.